src/Tools/dotnet-dev-certs/src/Program.cs

```diff
@@ -425,6 +425,10 @@ private static int EnsureHttpsCertificate(CommandOption exportPath, CommandOptio
             case EnsureCertificateResult.ErrorExportingTheCertificate:
                 reporter.Warn("There was an error exporting the HTTPS developer certificate to a file.");
                 return ErrorExportingTheCertificate;
+            case EnsureCertificateResult.ErrorExportingTheCertificateToNonExistentDirectory:
+                // A distinct warning is useful, but a distinct error code is probably not.
+                reporter.Warn("There was an error exporting the HTTPS developer certificate to a file.  Please create the target directory before exporting.");
```

Comment:
super nit: Single space after full stop.

Should we include the directory path and information about being careful about permissions?

https://github.com/dotnet/aspnetcore/pull/56985/files#diff-e47dcc2e277aacb04252a4182c94d046c117fca5a09b66228564fb796cdf547bR331

Reply:
Path was non-trivial to propagate; permissions felt too verbose, but I strongly considered it.

Reply:
```suggestion
                reporter.Warn("There was an error exporting the HTTPS developer certificate to a file. Please create the target directory before exporting. Choose permissions carefully when creating it.");
```